tests/014_print_complex_objects.ts

```diff
@@ -0,0 +1,2 @@
+console.log(["foo", "bar"]);
+console.log({"foo": "bar"});
```

Comment:
Could you use `js/unit_tests.ts` instead of `tests/`? ... these are relatively expensive to run.

Reply:
That's no problem. How would I capture `console.log`'s output in `js/unit_tests.ts`? I can't see a example that does that. Should I just call stringify directly?

Reply:
Yes, just do a unit test on stringify. I think the other console tests in unit_tests are not currently proper unit tests - they should be changed - but that'll be another commit.